### ISSUE TITLE
Add discoverability alerts for finding charges on sweater

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/balloons-and-static-electricity#168]
Design document provided by @terracoda outlines discoverability alerts for adding alerts to describe where more charges can be found on the sweater.

https://docs.google.com/spreadsheets/d/1BiXFN2dRWfsjqV2WvKAXnZFhsk0jCxbnT0gkZr_T5T0/edit?ts=568067c0#gid=232870371

This issue will document related changes and additions.

**Comment by jessegreenberg** [comment]
The descriptions for additional charge locations are:

- higher up
- further down
- further right
- further left

@terracoda will we need to handle cases where charges remain diagonally up/down/left/right from the balloon?

Also, how should the search algorithm work?  Should it just find the closest charge to the balloon and describe that?  Or should it prioritize a dense area of charges first?

For example, consider the following cases:
![config1](https://cloud.githubusercontent.com/assets/6396244/19084344/66883a32-8a23-11e6-808e-42839c691eec.PNG)

![config2](https://cloud.githubusercontent.com/assets/6396244/19084364/855dbd56-8a23-11e6-9d13-e2166f140069.PNG)

For scenarios like this, which charges should be described?

**Comment by terracoda** [comment]
@jessegreenberg, great questions! I was wondering the same :-) Goal of course, is to provide the most useful information with as few words as possible. I was thinking that describing the path to remaining charges, kind of like bread crumbing. But I still need to think about what pile to go after first.

**Comment by jessegreenberg** [comment]
Discussed with @terracoda, we are going to try something like:

- If person is moving in direction of charges, continue to guide in that direction
- Otherwise guide them to the location of the next closest charge

**Comment by jessegreenberg** [comment]
I am not sure why, but positions of charges in the sweater are stored in a propertyset.  They are also stored in an array.  Perhaps at some point we considered adding or removing these charges rather than toggling their visibility.

At any rate, this array of locations should be removed from the property set.

**Comment by terracoda** [comment]
@jessegreenberg yes, if the user is going in a direction that contains pairs of charges, the description can keep guiding in that direction, otherwise guide towards the closest pair of charges. I updated the design document to include descriptions in 2 directions. If two directions are needed to guide towards the closest pair of charges, start with up or down first. New description strings in document are:
- further up
- further down
- further right
- further left

And for two directions:
- up and to the right
- up and to the left
- down and to the right
- down and to the left

Example:
No change in charges. On left side of sweater. More pairs of charges further up and to the right.

**Comment by jessegreenberg** [comment]
Thanks @terracoda, I took a first pass at this.  I like the addition of two directions.  So far, it will only tell the user about the closest charges.

> if the user is going in a direction that contains pairs of charges, the description can keep guiding in that direction,

This part hasn't been implemented yet.

**Comment by jessegreenberg** [comment]
Removing high priority label, I would like to get more descriptions in place before adding the second part of this.

**Comment by terracoda** [comment]
@jessegreenberg sounds good to me!

**Comment by jessegreenberg** [comment]
Issue is stale, this has been well fleshed out in design documents.
